### DISCUSSION TITLE
How do I run ChatSSM and the openai.ipynb?

### DISCUSSION THREAD

**Discussion by mikheladam** [aitomatic/openssa#25] [Q&A]
Greetings! 

I am a beginner exploring AI so my technical skills are still in development. I have some problem running the openai.ipynb and the chatssm example. For the openai.ipynb, I've set my API key but when i try to run it, I get an error. Below is the code and corresponding error.

Code:
ssm.discuss("Remember, my name is CTN. I am a chatbot."), I get this error

Error:
InvalidRequestError: Must provide an 'engine' or 'model' parameter to create a <class 'openai.api_resources.chat_completion.ChatCompletion'>

I'm not sure where to put the parameter mentioned in the error.

Another error I'm getting is when I run the chatssm example and try to enter a prompt, I get the following reply:
"SYSTEM: HTTP error status: 500"

I can't find anything in the documentation that may help solve these. Apologies if these are very basic questions or if I've missed an important obvious step. Any help would be much appreciated.

**Comment by ctn** [comment]
This appears to be an old example (referring to `openssm`). @TheVinhLuong102 can you take a look? I’ll create an issue from this. Thanks @mikheladam .

**Comment by ctn** [comment]
See #51
